test_requirements.txt

```diff
@@ -6,6 +6,8 @@ coverage==4.4.2
 flake8==3.5.0
 psycopg2==2.7.3
 pylint==1.7.4
+pytest>=3.0.0; python_version>="3.4"
```

Comment:
Please remove `pytest` from here. Explanation at the top of the file.

Reply:
@dargueta  I have to leave it in there for tests to pass. Tox installs test_requirements first, and if pytest isn't pinned properly, the test_requirements installation will fail when it hits `pytest-mock` since it pulls in the latest version of pytest. I updated the comments in the file to address this